platformio.ini

```diff
@@ -8,17 +8,22 @@
 ; Please visit documentation for the other options and examples
 ; https://docs.platformio.org/page/projectconf.html
 
+[platformio]
+src_dir = firmware/src
+lib_dir = firmware/lib
+include_dir = firmware/include
+
 [env:esp32doit-devkit-v1]
 platform = espressif32
 board = esp32doit-devkit-v1
-board_build.embed_files =
-	icons/moonCloud.jpg
-	icons/sunClouds.jpg
-	icons/sun.jpg
-	icons/moon.jpg
-	icons/snow.jpg
-	icons/rain.jpg
-	icons/clouds.jpg
+board_build.embed_files = 
+	icons/jpg white/moonCloud.jpg
```

Comment:
@brett-dot-tech I have updated these to use the images that were already present in your repository (one level above the old `Info-orbs`)
Due to how this embedding works, the `icons.h` also needed updating to reflect these paths.